envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/GlobalSnapshot.kt

```diff
@@ -18,7 +19,8 @@ fun globalSnapshot(
     endpoints: Iterable<ClusterLoadAssignment> = emptyList(),
     properties: OutgoingPermissionsProperties = OutgoingPermissionsProperties(),
     clusterConfigurations: Map<String, ClusterConfiguration> = emptyMap(),
-    securedClusters: List<Cluster> = emptyList()
+    securedClusters: List<Cluster> = emptyList(),
+    tags: Map<String, Set<String>>
```

Comment:
You mean to use `typealias`?

Reply:
Yes or value/data class

Reply:
I think right know will be enough typealias to avoid boxing. If in the future we will need to add some functionality then it can be changed to value/data class